plugins/inputs/system/ps.go

```diff
@@ -73,35 +73,27 @@ func (s *systemPS) DiskUsage(
 	var usage []*disk.UsageStat
 	var partitions []*disk.PartitionStat
 
-	for i := range parts {
-
-		p := parts[i]
+	for _, p := range parts {
+		// If the mount point is not a member of the filter set,
+		// don't gather info on it.
+		if _, ok := mountPointFilterSet[p.Mountpoint]; !ok {
+			continue
+		}
```

Comment:
Ah, that's right. I think I was testing locally with non empty config. I'll fix that and update the PR.